cni-plugin/deployment/scripts/install-cni.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
```

Comment:
afaik in debian sh is symlinked to `dash` -- changed to `bash` for some extra goodies, such as regex operator in if conditions.